packages/airnode-protocol/contracts/rrp/authorizers/admin/interfaces/IAirnodeFeeRegistry.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+interface IAirnodeFeeRegistry {
+    event SetAirnodeEndpointFlag(
+        address airnode,
+        bool status,
+        address airnodeFlagAndPriceSetter
+    );
+
+    event SetDefaultPrice(uint256 price, address globalDefaultPriceSetter);
+
+    event SetDefaultChainPrice(
+        uint256 chainId,
+        uint256 price,
+        address globalDefaultPriceSetter
+    );
+
+    event SetDefaultAirnodePrice(
+        address airnode,
+        uint256 price,
+        address airnodeFlagAndPriceSetter
+    );
+
+    event SetDefaultChainAirnodePrice(
+        uint256 chainId,
+        address airnode,
+        uint256 price,
+        address airnodeFlagAndPriceSetter
+    );
+
+    event SetAirnodeEndpointPrice(
+        address airnode,
+        bytes32 endpointId,
+        uint256 price,
+        address airnodeFlagAndPriceSetter
+    );
+
+    event SetChainAirnodeEndpointPrice(
+        uint256 chainId,
+        address airnode,
+        bytes32 endpointId,
+        uint256 price,
+        address airnodeFlagAndPriceSetter
+    );
+
```

Comment:
We are missing all the view fuctions for the public state vars here and we should add override to all them after adding these functions.

Reply:
I've added them but do we normally implement view functions for state vars aren't they alreadt declared when declaring a public variable?

Reply:
It's something I started seeing being done by Burak in the Access Control Registry implementation.

Reply:
I guess we should do the same for IAirnodeTokenLock.sol

Reply:
Ill have to get back on this

Reply:
If you import an interface to another contract and the public state variable is not declared as a view function in the interface, you won't be able to read them